app/models/affiliate.rb

```diff
@@ -436,6 +454,20 @@ def validate_font_family(hash)
     errors.add(:base, 'Font family selection is invalid') if hash['font_family'].present? && !FontFamily.valid?(hash['font_family'])
   end
 
+  def validate_visual_design_json
+    return if visual_design_json.blank?
+
+    validate_visual_design_font_family(visual_design_json)
+  end
+
+  def validate_visual_design_font_family(visual_design_json)
+    %w[header_links_font_family footer_and_results_font_family].each do |font|
```

Comment:
Would it make sense to move these values to a constant?

Reply:
Good plan, doing.